Gemfile

```diff
@@ -16,9 +16,9 @@ gem "kubeclient", :git => "https://github.com/abonas/kubeclient", :branch => "ma
 gem "manageiq-loggers", "~> 0.4.0", ">= 0.4.2"
 gem "manageiq-messaging", "~> 0.1.2"
 gem "sources-api-client", "~> 1.0"
-gem 'topological_inventory-api-client',         "~> 2.0"
-gem "topological_inventory-ingress_api-client", "~> 1.0"
-gem "topological_inventory-providers-common", "~> 0.1"
+gem 'topological_inventory-api-client',         "~> 3.0"
```

Comment:
@mkanoor is there an issue with the topological_inventory-api-client v3.0 with the operationId? Or was that fixed manually?